### ISSUE TITLE
No device assignments were active

### ISSUE THREAD

**Issue by xiaomengxin123** [microsoft/tensorflow-directml#365]
When I use on windows10, just run a complex model ,it failed, the error like this
`No device assignments were active during op 'Embedding-Token/embedding_lookup/Identity' creation.`, others are used correctly.

**Comment by PatriceVignola** [comment]
Hi @xiaomengxin123, we'll need more information to diagnose the issue. What kind of model are you trying to run? Can you repro the error while using the vanilla TensorFlow 1.15 repo with a GPU device? It would help if you could share a script to repro the issue.

**Comment by xiaomengxin123** [comment]
Thanks and I see,the detail is 
`ReadVariableOp (ReadVariableOp) /device:DML:0 framework assigned device=/job:localhost/replica:0/task:0/device:DML:0
  VarIsInitializedOp (VarIsInitializedOp) /device:DML:0 framework assigned device=/job:localhost/replica:0/task:0/device:DML:0
  training/Adam/gradients/Embedding-Token/embedding_lookup_grad/Const (Const) /device:DML:0
  training/Adam/gradients/Embedding-Token/embedding_lookup_grad/Size (Size) /device:DML:0
  training/Adam/gradients/Embedding-Token/embedding_lookup_grad/ExpandDims/dim (Const) /device:DML:0
  training/Adam/gradients/Embedding-Token/embedding_lookup_grad/ExpandDims (ExpandDims) /device:DML:0
  training/Adam/gradients/Embedding-Token/embedding_lookup_grad/strided_slice/stack (Const) /device:DML:0
  training/Adam/gradients/Embedding-Token/embedding_lookup_grad/strided_slice/stack_1 (Const) /device:DML:0
  training/Adam/gradients/Embedding-Token/embedding_lookup_grad/strided_slice/stack_2 (Const) /device:DML:0
  training/Adam/gradients/Embedding-Token/embedding_lookup_grad/strided_slice (StridedSlice) /device:DML:0
  training/Adam/gradients/Embedding-Token/embedding_lookup_grad/concat/axis (Const) /device:DML:0
  training/Adam/gradients/Embedding-Token/embedding_lookup_grad/concat (ConcatV2) /device:DML:0
  training/Adam/gradients/Embedding-Token/embedding_lookup_grad/Reshape (Reshape) /device:DML:0
  training/Adam/gradients/Embedding-Token/embedding_lookup_grad/Reshape_1 (Reshape) /device:DML:0
  training/Adam/gradients/AddN_296/strided_slice/stack (Const) /device:DML:0
  training/Adam/gradients/AddN_296/strided_slice/stack_1 (Const) /device:DML:0
  training/Adam/gradients/AddN_296/strided_slice/stack_2 (Const) /device:DML:0
  training/Adam/gradients/AddN_296/strided_slice (StridedSlice) /device:DML:0
  training/Adam/gradients/AddN_296/inputs_1 (UnsortedSegmentSum) /device:DML:0
  training/Adam/gradients/AddN_296 (AddN) /device:DML:0
  training/Adam/ReadVariableOp_6 (ReadVariableOp) /device:DML:0
  training/Adam/AssignVariableOp_2 (AssignVariableOp) /device:DML:0
  training/Adam/ReadVariableOp_9 (ReadVariableOp) /device:DML:0 
[[node Embedding-Token/embedding_lookup/Identity (defined at \ProgramData\Anaconda3\envs\py37\lib\site-packages\tensorflow_core\python\framework\ops.py:1762) ]]Additional information about colocations:Node-device colocations active during op 'Embedding
-Token/embedding_lookup/Identity' creation:
  with tf.colocate_with(Embedding-Token/embeddings): <G:\ProgramData\Anaconda3\envs\py37\lib\site-packages\tensorflow_core\python\ops\embedding_ops.py:133>
No device assignments were active during op 'Embedding-Token/embedding_lookup/Identity' creation.
`
, Maybe it was caused by tensorflow_core and not model .
